tests/unit/test_xblock_poll.py

```diff
@@ -1,5 +1,6 @@
 import unittest
 import json
+
```

Comment:
@bradmerlin please undo this change since it doesnt add anything.

Reply:
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Oh nvm, I see Brad already mentioned that below.